packages/devextreme/js/__internal/scheduler/__tests__/legacy_form.test.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  beforeEach, describe, expect, it,
+} from '@jest/globals';
+
+import { createScheduler } from './__mock__/create_scheduler';
+import { setupSchedulerTestEnvironment } from './__mock__/m_mock_scheduler';
+
+describe('LegacyForm', () => {
+  beforeEach(() => {
+    setupSchedulerTestEnvironment();
+  });
+
+  it('should be disabled on default', async () => {
+    const { scheduler } = await createScheduler({ });
+
+    // @ts-expect-error private option
+    expect(scheduler.option('editing').legacyForm).toBe(false);
+  });
+
+  it('should be disabled on edit', async () => {
+    const { scheduler } = await createScheduler({ editing: { legacyForm: true } });
+
+    // @ts-expect-error private option
+    expect(scheduler.option('editing').legacyForm).toBe(true);
+  });
+
+  it('should can changed by option', async () => {
```

Comment:
typo in test name. Let's rename it to 'should be changed by option()' or 'is possible to update in runtime'